index.js

```diff
@@ -57,7 +60,12 @@ function forwardResponseToApiGateway(server, response, context) {
 
             Object.keys(headers)
                 .forEach(h => {
-                    if(Array.isArray(headers[h])) headers[h] = headers[h].join(',')
+                    if(Array.isArray(headers[h])) {
+                      headers[h].forEach((value, i) => {
+                        headers[binarycase(h, i + 1)] = value
```

Comment:
Definitely major version bump. We don't want to break existing users. I'll bump and release to NPM after merge.